code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -471,4 +471,8 @@
 		else
 			burn()
 
-	return shredded
\ No newline at end of file
+	return shredded
+
+/mob/living/carbon/human/proc/equipOutfit(outfit)
+	var/datum/outfit/O = new outfit
+	O.equip(src)
```

Comment:
Maybe also have support for passing an actual datum so later we can like make a datum and save all the clothes to it for reuse later.